mkt/lookup/forms.py

```diff
@@ -106,8 +107,8 @@ def save(self, obj, commit=True):
             destination = os.path.join(dirname, '%s' % obj.id)
 
             remove_promo_imgs(destination)
-            resize_promo_imgs(upload_path, destination,
-                              mkt.PROMO_IMG_SIZES, set_modified_on=[obj])
+            resize_promo_imgs(upload_path, destination, mkt.PROMO_IMG_SIZES)
```

Comment:
Now that you are no longer passing the `obj` here it won't update the hash correctly. You want to check the return value and use it to update the obj or call it async + chain w/ `update_from_data`